chore(docs): clarify API usage (#18926)
gatsbyjs/gatsby: docs/tutorial/part-seven/index.md

Stop and restart the development server. As you do, you'll see quite a few newly
created nodes get logged to the terminal console.

> You can use this API to add the slugs for your markdown pages to `MarkdownRemark`

Reviewer: Maybe something like: "In next section you will use this API [...]"?

Reviewer 2: ```suggestion
In next section you will use this API to add slugs for your Markdown pages to `MarkdownRemark`
```

> In next section you will use this API to add slugs for your Markdown pages to `MarkdownRemark`

Reviewer 2: ```suggestion
In the next section, you will use this API to add slugs for your Markdown pages to `MarkdownRemark`
```
